packages/nodes-base/nodes/Kafka/KafkaTrigger.node.ts

```diff
@@ -56,6 +58,28 @@ export class KafkaTrigger implements INodeType {
 				placeholder: 'n8n-kafka',
 				description: 'ID of the consumer group.',
 			},
+			{
+				displayName: 'Use Schema Registry',
+				name: 'useSchemaRegistry',
+				type: 'boolean',
+				default: false,
+				description: 'Use Apache Avro serialization format and Confluent\' wire formats.',
+			},
+			{
+				displayName: 'Schema Registry URL',
+				name: 'schemaRegistryUrl',
+				type: 'string',
+				required: true,
+				displayOptions: {
+					show: {
+						useSchemaRegistry: [
+							true,
+						],
+					},
+				},
+				default: '',
```

Comment:
should be nice to add a `placeholder` value, to let the user know the default port (8081) of the schema registry
something. (the same in Kafka node)

Reply:
done